### ISSUE TITLE
Schema Generate problem with reference tables

### ISSUE THREAD

**Issue by AliSayyah** [piccolo-orm/piccolo#259]
- schema name doesn't get included in `tablename`.
- reference tables are written before import statements.
- only referenced column gets added.

```python
class MyTable(Table):
    id = Serial(null=False, primary_key=True, unique=False, index=False, index_method=IndexMethod.btree, choices=None)

from piccolo.columns.base import OnDelete
from piccolo.columns.base import OnUpdate
from piccolo.columns.column_types import BigInt
from piccolo.columns.column_types import Boolean
from piccolo.columns.column_types import DoublePrecision
from piccolo.columns.column_types import ForeignKey
from piccolo.columns.column_types import Integer
from piccolo.columns.column_types import Serial
from piccolo.columns.column_types import Text
from piccolo.columns.column_types import Varchar
from piccolo.table import Table
```

I think referenced tables should be treated as other tables.

**Comment by dantownsend** [comment]
Hmm, that's odd. I can't replicate this locally.

If should declare the referenced table properly.

There are two possible reasons I can think of:

 * Some other piece of code is printing out the `MyTable` definition, so when `piccolo schema generate` is being piped to a file, it includes this extra content? It might be worth calling `get_output_schema` directly to verify this.
 * A strange edge case where a referenced table can't be found in the database?

**Comment by AliSayyah** [comment]
@dantownsend I checked and your first guess is correct.
how can I prevent this?

**Comment by dantownsend** [comment]
@AliSayyah I encountered similar issues when originally writing it. The issue was when Piccolo was printing out warnings. I managed to solve those by using `colored_warning` instead of `print`.  I think that warnings get written to stderr instead of stdout, so doesn't get piped to the file.

I guess the first thing is isolating where the `MyTable` definition is getting printout out, and deciding if it's a warning, which can use `colored_warning` instead.
